tests/UKHO.ERPFacade.API.UnitTests/Helpers/EncContentSapMessageBuilderTests.cs

```diff
@@ -25,10 +26,13 @@ public class EncContentSapMessageBuilderTests
         private IFileSystemHelper _fakeFileSystemHelper;
         private IOptions<SapActionConfiguration> _fakeSapActionConfig;
         private ILogger<EncContentSapMessageBuilder> _fakeLogger;
+        private IPermitDecryption _fakePermitDecryption;
 
         private EncContentSapMessageBuilder _fakeEncContentSapMessageBuilder;
         private const string XpathActionItems = $"//*[local-name()='ACTIONITEMS']";
         private const string EncCell = "ENC CELL";
+        private readonly string XpathActiveKey = $"//*[local-name()='ACTIVEKEY']";
+        private readonly string XpathNextKey = $"//*[local-name()='NEXTKEY']";
 
         private readonly string sapXmlFile = @"<?xml version=""1.0"" encoding=""utf-8""?>
```

Comment:
Can we make this variable as constant please. Also do the same for other applicable variables.